notebooks/05-operations/02-druid-logs-asynch.ipynb

```diff
@@ -0,0 +1,381 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c9fc0614-4eec-4c39-ad22-ea10afa00e9d",
+   "metadata": {},
+   "source": [
+    "# Streaming and SQL-based ingestion logs\n",
+    "\n",
+    "Apache Druid uses Log4J to emit information as it runs. They not only enable you to investigate issues and solve problems, but to understand how each of Druid's processes work in isolation and in collaboration with one another.\n",
+    "\n",
+    "As well as the usual processes in Druid, Druid's highly-parallelised approach to both [streaming](https://druid.apache.org/docs/latest/ingestion/#streaming) and [batch](https://druid.apache.org/docs/latest/ingestion/#batch) ingestion is carried out by worker tasks. Each of these is a separate process that create their own log files.\n",
+    "\n",
+    "In this notebook, you will look at log files for the key processes in SQL-based ingestion as well as the spawned peon processes, and see where to look to point the all-important task log files to durable storage."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e8dbaa49-8368-45cd-82da-2e358d17fdc4",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "This tutorial works with Druid 28.0.1 or later.\n",
+    "\n",
+    "### Run with Docker\n",
+    "\n",
+    "Launch this tutorial and all prerequisites using the `jupyter` profile of the Docker Compose file for Jupyter-based Druid tutorials. For more information, see the Learn Druid repository [readme](https://github.com/implydata/learn-druid)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "151ede96-348f-423c-adc7-0e23c6910b9e",
+   "metadata": {},
+   "source": [
+    "## Initialization\n",
+    "\n",
+    "To use this notebook, you will need to run Druid locally.\n",
+    "\n",
+    "You will also make extensive use of the terminal, which we suggest you place alongside this notebook or on another screen.\n",
```

Comment:
"which we suggest you place alongside this notebook"
suggest
"which you can place alongside this notebook"